rust/utils/run-locally/src/invariants.rs

```diff
@@ -65,12 +65,16 @@ pub fn termination_invariants_met(
     const STORING_NEW_MESSAGE_LOG_MESSAGE: &str = "Storing new message in db";
     const LOOKING_FOR_EVENTS_LOG_MESSAGE: &str = "Looking for events in index range";
     const HYPER_INCOMING_BODY_LOG_MESSAGE: &str = "incoming body completed";
+
+    const TX_ID_INDEXING_LOG_MESSAGE: &str = "Found log(s) for tx id";
```

Comment:
curious if you think we should use a shared const in the log site too. Don't feel strongly

Reply:
I think this log would live in `hyperlane-base` and somehow that might create a circular dependency

Reply:
I don't think hyperlane-base depends on run-locally, but not a big deal

Reply:
or is it that other crates would then need to import from hyperlane-base

Reply:
turned out it works to just define in it `hyperlane-base`, don't know why I remember it not working